cmd/assetsvc/handler.go

```diff
@@ -100,14 +103,14 @@ func min(a, b int) int {
 }
 
 func getPaginatedChartList(namespace, repo string, pageNumber, pageSize int) (apiListResponse, interface{}, error) {
-	charts, totalPages, err := manager.getPaginatedChartList(namespace, repo, pageNumber, pageSize)
-	return newChartListResponse(charts), meta{totalPages}, err
+	charts, totalPages, totalCharts, err := manager.getPaginatedChartList(namespace, repo, pageNumber, pageSize)
```

Comment:
why is the totalCharts number necessary?

Reply:
It is not strictly necessary at all. In fact, one can roughly estimate this number by size*totalPages. However, if, in the future, we want to display the total amount of results in the UI, we will have this number directly in the meta information.
But, as you pointed out, currently it is not being useful.

Reply:
okay, so simplicity matters, I would remove this. The less we need to maintain the better :)

Reply:
As requested, removed totalChart; simplicity wins.
If, eventually, we want to get it back again, we will just have to revert 
the commit a2e0e4cea35c05838cd0f4db003ba5529d5cc48a